metadata/metadata.go

```diff
@@ -244,12 +217,14 @@ func updateEtag(resp *http.Response) bool {
 	return etag != oldEtag
 }
 
-func watchMetadata(ctx context.Context) (*metadata, error) {
-	return getMetadata(ctx, true)
+// Watch runs a longpool on metadata server.
+func Watch(ctx context.Context) (*Descriptor, error) {
+	return Get(ctx, true)
 }
 
-func getMetadata(ctx context.Context, hang bool) (*metadata, error) {
-	logger.Debugf("getMetadata, %t", hang)
+// Get does a metadata call, if hang is set to true then it will do a longpool.
+func Get(ctx context.Context, hang bool) (*Descriptor, error) {
+	logger.Debugf("Get, %t", hang)
```

Comment:
this will log just `Get, true/false` can we log something like `Invoking Get metadata, wait for change: true`

Reply:
Harmless, I'll have it done for this PR. Thank you.